x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -87,25 +112,39 @@ keys must have corresponding length according to the following table:
 [discrete]
 ==== Password Hashing
 
-{es} offers a number of algorithms for securely hashing credentials in memory and
-on disk. However, only the `PBKDF2` family of algorithms is compliant with FIPS
-140-2 for password hashing. You must set the `cache.hash_algo` realm settings
+While {es} offers a number of algorithms for securely hashing credentials in memory and
+on disk, only the `PBKDF2` based family of algorithms is compliant with FIPS
+140-2 for password hashing. However, since `PBKDF2` is essentially a key derivation
+function, your JVM security provider may enforce a 112 bit key strength requirement
+(see <<keystore-fips-password>>), and this requirement may affect password hashing
+in {es}, even though FIPS 140-2 does not mandate password standards for user
+authentication. In order to accommodate this requirement, 
+while allowing you to use passwords that satisfy your security policy, {es} offers
```

Comment:
Streamlines a bit to remove the parenthetical.
```suggestion
function, your JVM security provider may enforce a
<<keystore-fips-password,112-bit key strength requirement>>. Although FIPS 140-2
does not mandate user password standards, this requirement may affect password
hashing in {es}. To comply with this requirement and your security policy, {es} offers
```

Reply:
I used a variation of the suggestion, thanks!